app/viewsapi/FormFunctions.scala

```diff
@@ -13,6 +16,28 @@ object FormFunctions {
       if (args.exists(_ == (requiredInputArg, true))) "*" else ""
     }
   }
+  class ErrorHandling(elements: FieldElements) {
+
+    def setErrorClass(): String = {
+      if(elements.hasErrors) {
+        "govuk-form-group--error"
+      } else {
+        ""
+      }
+    }
+
+    def renderErrorMessage(): Html = {
+      if(elements.hasErrors) {
+        Html("<span id=\"error\" class=\"govuk-error-message\"><span class=\"govuk-visually-hidden\">Error:</span>" +
```

Comment:
Is there any way to move this to the templates, so we don't have to hard-code HTML in a Scala object?